docs/documentation/maya/simple_setup.md

```diff
@@ -110,12 +135,12 @@ Once we have the Connection Editor open we must first select the locator from ou
 
 <figure>
   <img src="images/setup_muscle_8.png"> 
-  <figcaption>Figure: Connection Editor tool.</figcaption>
+  <figcaption>Figure 15: Connection Editor tool.</figcaption>
 </figure>
 
 Now we will see that when the elbow gets flexed (and therefore the angle from the locator gets smaller) the muscle activation will get higher, simulating a much more realistic scenario.
 
-AdonisFX provides an endless array of customization and tweaking to fine tune the muscle dynamics by making use of these tools and the many additional parameters of the AdnMuscle deformer, that you can consult [here](muscle.md).
+AdonisFX provides an endless array of customization and tweaking to fine tune the muscle dynamics by making use of these tools and the many additional parameters of the AdnMuscle deformer, that you can consult in the documentation for [AdnMuscle](muscle.md).
```

Comment:
We should be careful with this. We are in docs not marketing. Please rephrase this without saying things like "endless array of customization and tweaking"

Reply:
With that being said we might want to expose an example for setting up slide on segment constraints

Reply:
The rephrasing was already addressed.